Integrations/integration-Cloaken.yml

```diff
@@ -0,0 +1,79 @@
+description: 'Unshorten urls through a tor proxy server. Unshorten urls onsite with
```

Comment:
Please move this `Created by CypherInt. Visit us at https://cypherint.com. Subscribe on the Amazon marketplace: https://aws.amazon.com/marketplace/` to the detailed description

Reply:
ok done